learning/katas/go/Core Transforms/Map/ParDo/pkg/task/task.go

```diff
@@ -18,8 +18,9 @@ package task
 import "github.com/apache/beam/sdks/go/pkg/beam"
 
 func ApplyTransform(s beam.Scope, input beam.PCollection) beam.PCollection {
-	processFn := func(element int) int {
-		return element * 10
-	}
-	return beam.ParDo(s, processFn, input)
+	return beam.ParDo(s, multiplyBy10Fn, input)
 }
+
+func multiplyBy10Fn(element int) int {
+	return element * 10
```

Comment:
The "* 10" is not covered by the answer placeholder. Is it intentional?

![image](https://user-images.githubusercontent.com/5459430/80789287-24764800-8bbe-11ea-9594-d279d633e9e7.png)

Reply:
What's odd is that I saw the following when testing the course in both IntelliJ and Goland:

<img width="647" alt="Screen Shot 2020-05-03 at 11 25 01 AM" src="https://user-images.githubusercontent.com/762456/80922441-092c4980-8d32-11ea-9796-a8d0e0ede524.png">

Reply:
This looks good now